src/bandersnatch/package.py

```diff
@@ -46,15 +45,11 @@ def __init__(
         name: str,
         serial: Union[int, str],
         mirror: "Mirror",
-        *,
-        cleanup: bool = False,
     ) -> None:
         self.name = canonicalize_name(name)
         self.raw_name = name
-        self.normalized_name_legacy = utils.bandersnatch_safe_name(name)
```

Comment:
Want to keep legacy things out of the public API if possible, which is easy because this field was *only* used for cleanup